compose-app/src/main/java/com/mapbox/maps/extension/compose/MapboxMapScope.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.maps.extension.compose
+
+import androidx.compose.foundation.layout.LayoutScopeMarker
+import androidx.compose.runtime.Immutable
+import com.mapbox.maps.MapSurface
+import com.mapbox.maps.MapView
+
+@LayoutScopeMarker
+@Immutable
+public interface MapboxMapScope {
```

Comment:
should we specify allowed functions? Or user can add any composable code?

Reply:
This can be done with a separate annotation. MapboxMapScope can only provide extensions.